common/sentinel/src/config/config.rs

```diff
@@ -46,7 +46,7 @@ pub struct SentinelConfig {
     ipfs: IpfsConfig,
     core: SentinelCoreConfig,
     governance: GovernanceConfig,
-    networks: HashMap<NetworkId, NetworkConfig>,
+    pub networks: HashMap<NetworkId, NetworkConfig>,
```

Comment:
The struct derives `derive_getters::Getters`, so you can just call `sentinel_config.networks()` to access the value!